scripts/userstyles.yml

```
@@ -429,6 +431,13 @@ userstyles:
     readme:
       app-link: https://news.ycombinator.com
     current-maintainers: [*lucasmelin]
+  hardcover:
+    name: Hardcover
+    categories: [library, social_networking]
+    color: mauve
```

```suggestion
    color: lavender
```
Feels more appropriate.